<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="INLN34364442A" failures="3" tests="3" name="tests.VerifyCompanyVoteCardTest" time="82.107" errors="0" timestamp="2021-02-22T12:09:33 IST" skipped="0">
  <testcase classname="tests.VerifyCompanyVoteCardTest" name="verifyCompanyVoteCardTest" time="37.171">
    <failure message="Company Name Activision does not exist" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Company Name Activision does not exist
at org.testng.Assert.fail(Assert.java:99)
at utils.GenericUtils.checkFieldExist(GenericUtils.java:181)
at pages.VerifyCompanyVoteCardPage.checkCompanyNameBanner(VerifyCompanyVoteCardPage.java:81)
at tests.VerifyCompanyVoteCardTest.verifyCompanyVoteCardTest(VerifyCompanyVoteCardTest.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- verifyCompanyVoteCardTest -->
  <system-out/>
  <testcase classname="tests.VerifyCompanyVoteCardTest" name="verifyCompanyVoteCardTest" time="19.297">
    <failure message="Click on Items per page arrow icon failed" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Click on Items per page arrow icon failed
at org.testng.Assert.fail(Assert.java:99)
at utils.GenericUtils.click(GenericUtils.java:109)
at pages.VerifyCompanyVoteCardPage.clickItemsPerPageArrow(VerifyCompanyVoteCardPage.java:59)
at tests.VerifyCompanyVoteCardTest.verifyCompanyVoteCardTest(VerifyCompanyVoteCardTest.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- verifyCompanyVoteCardTest -->
  <system-out/>
  <testcase classname="tests.VerifyCompanyVoteCardTest" name="verifyCompanyVoteCardTest" time="25.639">
    <failure message="Click on Items per page arrow icon failed" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Click on Items per page arrow icon failed
at org.testng.Assert.fail(Assert.java:99)
at utils.GenericUtils.click(GenericUtils.java:109)
at pages.VerifyCompanyVoteCardPage.clickItemsPerPageArrow(VerifyCompanyVoteCardPage.java:59)
at tests.VerifyCompanyVoteCardTest.verifyCompanyVoteCardTest(VerifyCompanyVoteCardTest.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
]]>
    </failure>
  </testcase> <!-- verifyCompanyVoteCardTest -->
  <system-out/>
</testsuite> <!-- tests.VerifyCompanyVoteCardTest -->
